easybuild/easyconfigs/a/ASE/ASE-3.16.0-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'ASE'
+version = '3.16.0'
```

Comment:
not `3.16.2`?

Reply:
GPAW 1.4.0 says that 3.16.0 was the corresponding release:
https://wiki.fysik.dtu.dk/gpaw/releasenotes.html#releasenotes
I don't mind bumping to 3.16.2 if you guys think it would still work.